custom-functions/remove-null-fields/README.md

```diff
@@ -26,8 +26,12 @@ The `remove-null-fields` function, on the other hand, directly traverses each ke
 
 The benchmark sample below demonstrates that using the Parser Function with the built-in `Fields Filter Expression` to remove empty and null fields is more expensive compared to using the Parser Function without removal and the `remove-null-fields` function.
 
-- **Parser Function with Built-in Removal**: `60656.60ms`
-- **Parser Function + Remove Null Fields Function**: `2029.45ms`
+**Test System:**
```

Comment:
Added test system, so I can consistently compare benchmarks in the future, if needed.

Reply:
Updated the values below, to those from benchmarks on the test system